LICENSE

```diff
@@ -1,6 +1,6 @@
-This project is licensed under the MIT license.
+MIT License
 
-Copyright (c) 2013 - 2014 CocoaPods Dev Team
+Copyright (c) 2016 William Entriken
```

Comment:
And this one back, I'm happy to have this `CP Dev Team, and William Entriken` instead though